### PR TITLE
Add logo.svg, update index.html

### PR DESCRIPTION
**Pull request by DiegoEmilio01**

### CODE REVIEW

**Review by nbcl — CHANGES_REQUESTED** [review]
Hey @DiegoEmilio01!

Everything looks good.

I have added two comments about minor fixes that should be considered before merging.

File: css/rheviu.css [hunk 1]
```diff
@@ -3,6 +3,10 @@ body {
     overflow-x: hidden;
 }
 
+.logo {
```
Comment (nbcl) [inline css/rheviu.css]: Instead of implementing a new class, we could use `rounded` or `rounded-sm` to achieve the same effect.

File: index.html [hunk 2]
```diff
@@ -39,10 +39,7 @@
                 class="flex justify-between items-center border-b-2 border-gray-100 py-2 md:justify-start md:space-x-10">
                 <div class="flex justify-start lg:w-0 lg:flex-1">
                     <a href="/">
-                        <svg class="h-6 w-auto sm:h-6 mt-3" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
-                            <path fill="#0054FF"
-                                d="M2 2v13.5a.5.5 0 0 0 .74.439L8 13.069l5.26 2.87A.5.5 0 0 0 14 15.5V2a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2z" />
-                        </svg>
+                        <img class="logo h-6 w-auto sm:h-6 mt-3" src="logo.svg" loading="lazy" alt="Logo Rheviu">
```
Comment (nbcl) [inline index.html]: This addition should be mirrored in the footer, where there is another instance of a placeholder logo.

**Review by nbcl — CHANGES_REQUESTED** [review]
Hey @DiegoEmilio01!

Everything looks good.

I have added two comments about minor fixes that should be considered before merging.

File: css/rheviu.css (hunk 1 above)
Comment (DiegoEmilio01) [reply css/rheviu.css]: Done

File: index.html (hunk 2 above)
Comment (DiegoEmilio01) [reply index.html]: Done

**Comment by DiegoEmilio01** [comment]
Pushed the changes from a branch with the updates from the older (and merged) pull request so it doesn't throw conflicts @nbcl

**Review by nbcl — APPROVED** [review]

**Pull request merged**